# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/rene/fbset/install.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./Makefile.orig	2003-03-22 11:05:26.000000000 +0100
+++ ./Makefile	2003-03-22 11:06:04.000000000 +0100
@@ -25,6 +25,7 @@
 
 install:	fbset
 		if [ -f /sbin/fbset ]; then rm /sbin/fbset; fi
+		if [ -f /usr/sbin/fbset ]; then rm /usr/sbin/fbset; fi
-		$(INSTALL) fbset /usr/sbin
+		$(INSTALL) fbset /bin
 		$(INSTALL) fbset.8 /usr/man/man8
 		$(INSTALL) fb.modes.5 /usr/man/man5
